src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelUploaderTest.java

```diff
@@ -95,7 +96,8 @@ public void parseXlsxShouldReturnWorkbookObjectTest() throws IOException {
                 getMultipartFile("src/integrationTest/resources/CaseWorkerUserXlsxWithNoPassword.xlsx",
                         TYPE_XLSX));
         List<CaseWorkerProfile> profiles = excelAdaptorService.parseExcel(workbook, CaseWorkerProfile.class);
-        assertThat(profiles).hasSize(workbook.getSheetAt(1).getPhysicalNumberOfRows() - 1);
+        assertThat(profiles).hasSize(workbook.getSheet(CaseWorkerConstants.REQUIRED_CW_SHEET_NAME)
```

Comment:
Static import of `REQUIRED_CW_SHEET_NAME` would be nicer

Reply:
sure, will change :)